pyfstat/make_sfts.py

```diff
@@ -943,6 +941,175 @@ def __init__(
         )
 
 
+class TransientLineWriter(Writer):
```

Comment:
As discussed, I'd prefer a more general name. `LineWriter` or `UnmodulatedArtifactWriter`.

Reply:
What about
```
MonochromaticArtifactWriter
```
?

We are technically injecting a signal
```
h(t) = h0 * sin(Omega * t + Phi0)
```
where `h0 = ct.`, meaning it's as monochromatic as it can get.

Of course, that doesn't hold when we include transient lines (i.e. windowed lines), but `Unmodulated` doesn't either.

About the `Artifact` term... I don't have a strong opinion.

Reply:
I think I like `LineWriter` best, as the others are more technically precise, but this is established jargon that most active CWers will get immediately, and much shorter.

Also leaves open the possibility of generalizing it.

Reply:
I like the jargon argument. I'll update docs accordingly.

Reply:
89006f7